test/files/rules/indentwith_tabs.test.ts

```diff
@@ -1,5 +1,8 @@
 // valid code
 module TestModule {
+	/**
+	 * Some documentation
+	 */
```

Comment:
@gscshoyru thoughts on my example here? Maybe we should continue checking indentation within multiline comments, and only allow a single extra space before the `*`?